reladomo/src/main/java/com/gs/fw/common/mithra/portal/MithraAbstractObjectPortal.java

```diff
@@ -553,6 +555,11 @@ public MithraObjectReader getMithraObjectReader()
         return mithraObjectReader;
     }
 
+    public void setMithraObjectReader (MithraObjectReader mithraObjectReader)

```

Comment:
What is this for?

Reply:
This is included to support MithraPureObject deserialization